frontend-react/src/utils/UserUtils.ts

```diff
@@ -3,12 +3,15 @@ import { OktaAuth } from "@okta/okta-auth-js";
 function logout(oktaAuth: OktaAuth): void {
     if (oktaAuth?.signOut) {
         try {
-            oktaAuth.signOut();
+            oktaAuth.signOut().finally(() => {
+                const bc = new BroadcastChannel("session");
```

Comment:
Hmmmm now that I see this, can we just create ONE BroadcastChannel for the entire lifetime of the app?  That way, we won't have to worry about needing to unbind/close [up here](https://github.com/CDCgov/prime-reportstream/pull/7727/files#r1053662183), and we'll have one "global" instance to funnel our session events through.

Reply:
👍 Will find a place for the ref & its instantiation 🦖 . Perhaps the `AuthStateManager`'s data? 🤔

Reply:
In my mind, there are a couple of ways we can handle this:

// Method 1: instantiate as soon as JavaScript executes
// This naturally ensures that we'll only create one BroadcastChannel for the lifetime of the app,
// but it can be a little bit harder for test environments since it relies on a "global" (not really global, just not in the scope of any other function)
```js
// in UserUtils.ts
export const sessionBroadcastChannel = new BroadcastChannel("session")
sessionBroadcastChannel.addEventListener(...);

// then we can import sessionBroadcastChannel if we want to do anything else with it
// in SomeOtherFile.ts
import { sessionBroadcastChannel } from './UserUtils';
sessionBroadcastChannel.addEventListener(...);
sessionBroadcastChannel.close();
```

// Method 2: wrap the instantiation and close in functions
// This is safer, and we can add checks to prevent duplicity, but it requires a bit more code:
```js
// in UserUtils.ts

// initialize to null in the "global" scope
let sessionBroadcastChannel = null;

export function initializeSessionBroadcastChannel() {
  // don't reinitialize
  if (sessionBroadcastChannel) {
    return sessionBroadcastChannel;
  }

  sessionBroadcastChannel = new BroadcastChannel("session");
  return sessionBroadcastChannel;
}

export function closeSessionBroadcastChannel() {
  sessionBroadcastChannel?.close(); // close channel and mark for garbage collection
  sessionBroadcastChannel = null;
}

export function getSessionBroadcastChannel() {
  return sessionBroadcastChannel;
}

// we'll have to import and invoke these functions instead of the straight BroadcastChannel instance
// in SomeOtherFile.ts
import { initializeSessionBroadcastChannel } from './UserUtils';

broadcastChannel = initializeSessionBroadcastChannel();
broadcastChannel.addEventListener(...)
```

We could add other conveniences to respond to `login` and `logout` events automatically, but we gotta go to sprint kickoff now!  Happy to chat more about this later, though!  And sorry for the wall o' text 😆